twitter_tunes/scripts/youtube_api.py

```diff
@@ -0,0 +1,90 @@
+# coding=utf-8
+from apiclient.discovery import build
+from apiclient.errors import HttpError, UnknownApiNameOrVersion
+import os
+from httplib2 import ServerNotFoundError
+
+
+YOUTUBE_DEVELOPER_KEY = os.environ.get('YT_AUTH')
+YOUTUBE_API_SERVICE_NAME = 'youtube'
+YOUTUBE_API_VERSION = 'v3'
+TERMS = ['remix', 'music', 'song', 'parody', 'jam', 'dance']
+
+
+def youtube_search(keyword, max_results=20):
+    """Query YouTube API for search results based off keyword search."""
+    try:
+        youtube = build(
+            YOUTUBE_API_SERVICE_NAME,
+            YOUTUBE_API_VERSION,
+            developerKey=YOUTUBE_DEVELOPER_KEY
+            )
+        search_response = youtube.search(
+        ).list(
+                q=keyword + ' music',
+                type='video',
+                part='id,snippet',
+                maxResults=max_results).execute()
+        return search_response
+    except HttpError as err:
+        print('An HTTP error has occurred.  Please check YT Developer Key.')
+        return err
```

Comment:
odd.  I'm thinking re-raising rather than returning this error would be more productive and useful.